.github/ISSUE_TEMPLATE/Bug_Report.md

```diff
@@ -1,46 +1,45 @@
+---
+name: Bug report 🐛
+about: Report errors or unexpected behavior 🤔
+
+---
 <!--
 
 For Windows PowerShell 5.1 issues, suggestions, or feature requests please use the following link instead:
-- Windows PowerShell [UserVoice](https://windowsserver.uservoice.com/forums/301869-powershell)
+Windows PowerShell [UserVoice](https://windowsserver.uservoice.com/forums/301869-powershell)
+
+This repository is **ONLY** for PowerShell Core 6 issues.
 
-If it is a bug report:
-- make sure you are able to repro it on the latest released version. 
+- Make sure you are able to repro it on the latest released version.
 You can install the latest version from https://github.com/PowerShell/PowerShell/releases
```

Comment:
Lines 13-14 are about the same thing.
Probably make 'latest released version' in line 13 a hyperlink with url from line 14; and remove line 14.